Adds AI, telecomms, bridge consoles and some other misc. things to runtime for easier debugging

Amunak opened the pull request:
* Added a new secure crate preset (locked to ERT) with _every single AI module_.

Runtime-2 additions:
* Telecomms (general and engineering channel should work fully, a NTnet relay is also present). Includes cooling and consoles.
* AI core, AI upload with borg spawners and charger, turrets, blast doors. The upload also contains the aforementioned AI modules crate.
* Messaging server room.
* Added cameras all over the place (3 networks total, also added networks to map descriptor).
* Added some intercoms, holopads and displays (both statin and AI status).
* Added some important consoles to bridge.
* Mapped in the magical SMES instead of one of the RTGs - there's no way any number of RTGs would be able to power the tcomms (~75kW) and an AI (50kW).
* Added some newscasters and a bridge and AI requests console.
* Added powernet sensors for the 3 powernets we now have for the power console to be of any use.

This means we can now debug radio-related stuff, PDA messaging, anything AI/borg related.

Marked as `backend` in changelog not to scare people about changes to a map they won't ever see. Hope that is fine?

Amunak commented:
!review

JohnWildkins commented:
while you're at it would you mind mapping in a NTnet relay for modular computers

Amunak commented:
> while you're at it would you mind mapping in a NTnet relay for modular computers

Sure, done :)

fernerr reviewed (changes requested):
![flaps](https://user-images.githubusercontent.com/35879136/96365006-0b4a5f00-113e-11eb-9812-fb6cc9e73b62.PNG)
 - The central compartment could use some airflaps too.
 - The turret controller should be moved to be right next to the entrance, so it can be operated from both inside and outside the chamber, else poor bstechs will get stunned all over.

Amunak commented:
Fair points, changed. I also fixed badly mapped tcomms area after some last minute changes.

-----

The following additional changes are now reflected in the PR description as well:

Additionally I made the AI chamber smaller, added AI private channel intercoms, requests console, newscaster (and to the bridge, too). I also mapped in air pipes and a new crate with all AI modules.

I also noticed the RTGs could never power the base, so I added in a magical SMES in place of the middle one. Oh and I added some powernet sensors.
